client/src/test/scala/org/http4s/client/testroutes/GetRoutes.scala

```diff
@@ -35,6 +37,9 @@ object GetRoutes {
   def getPaths(implicit timer: Timer[IO]): Map[String, IO[Response[IO]]] =
     Map(
       SimplePath -> Response[IO](Ok).withEntity("simple path").pure[IO],
+      LargePath -> Response[IO](Ok).withEntity("a" * 8 * 1024).pure[IO],
+      HugePath -> IO( // delayed in IO to avoid allocating the huge string unless necessary
+        Response[IO](Ok).withEntity("a" * 100 * 1024 * 1024)),
```

Comment:
Do we ever do this under load where we might be allocating several of these at once?  It could be a pure stream, so we should be able to share this response and pay for it once.

Or could we just stream it?  Is the point to be a 100MB chunk, or is the point to return 100MB?

Reply:
For now we use it in only one test and the goal is to have a response large enough that if the client tried to wait for it, it would take more than 5 seconds. The test would then hit the 5 seconds timeout and we would know that something is wrong.

I think I could rename it to "infinite" and then make it an infinite stream. For that specific test it shouldn't matter. In fact it would be better because that would remove the assumption that the test implementation currently makes: "transfering 100MB takes more than 5 seconds".